rebar.config

```diff
@@ -4,6 +4,7 @@
 %% escript_incl_extra is for internal rebar-private use only.
 %% Do not use outside rebar. Config interface is not stable.
 {escript_incl_extra, [{"priv/templates/*", "."}]}.
+{escript_emu_args, "%%! -pa rebar/rebar/ebin +A0\n"}.
```

Comment:
Can you insert a blank line or alternatively add a comment explaining why we set `+A0`?